### PR TITLE
Switch to Actual LRPS BBS endpoint

### PR DESCRIPTION
**Pull request by mariash**
Switch to ActualLRPs BBS endpoint. 

Diego v2.16.0 introduced ActualLRPs endpoint to replace ActualLRPGroups (august 2018).
Diego v2.31.0 marks ActualLRPGroups as deprecated.

ActualLRPGroups endpoint is deprecated. ActualLRPs returns the list of all LRPs without grouping them by their presence. ActualLRPGroup used to contain Instance and Evacuating key, where one of them will be set. Now the list will contain all LRP.

* [X] I have reviewed the [contributing guide](https://github.com/cloudfoundry/cloud_controller_ng/blob/master/CONTRIBUTING.md)

* [X] I have viewed, signed, and submitted the Contributor License Agreement

* [X] I have made this pull request to the `master` branch

* [X] I have run all the unit tests using `bundle exec rake`

* [X] I have run [CF Acceptance Tests](https://github.com/cloudfoundry/cloud_controller_ng#cf-acceptance-tests-cats)

@cloudfoundry/cf-diego

### CODE REVIEW

File: spec/unit/lib/cloud_controller/diego/bbs_instances_client_spec.rb [hunk 1]
```diff
@@ -6,26 +6,19 @@ module VCAP::CloudController::Diego
...
-        allow(bbs_client).to receive(:actual_lrp_groups_by_process_guid).with(process_guid).and_return(bbs_response)
+        allow(bbs_client).to receive(:actual_lrps_by_process_guid).with(process_guid).and_return(bbs_response)
       end
 
       it 'sends the lrp instances request to diego' do
         client.lrp_instances(process)
-        expect(bbs_client).to have_received(:actual_lrp_groups_by_process_guid).with(process_guid)
-      end
-
-      it 'returns a resolved list of actual LRPs' do
```
Comment (selzoc) [inline spec/unit/lib/cloud_controller/diego/bbs_instances_client_spec.rb]: why was this test removed?

Comment (mariash) [reply spec/unit/lib/cloud_controller/diego/bbs_instances_client_spec.rb]: There is no resolving required any more, you get the list of LRPs not groups, so no need to convert them from groups. `lib/diego/actual_lrp_group_resolver.rb` is removed.

Comment (selzoc) [reply spec/unit/lib/cloud_controller/diego/bbs_instances_client_spec.rb]: cool, thanks!

**Comment by tcdowney** [comment]
Looks good, thanks @mariash!

**Comment by heyjcollins** [comment]
@ssisil - as per our conversation, I'm adding some validation steps I intended to go through in my AC.
* disclaimer - I'd recco reviewing these steps with a professional as a cross-check.

**deployment**
deploy a foundation that includes the PR'd change to capi-release and BBS Access Log enabled... 
Example ops file for log enablement: 
```
---
- type: replace
  path: /instance_groups/name=diego-api/jobs/name=bbs/properties/diego?/bbs?/enable_access_log?
  value: true
```

**steps**
1. note the ip address of the `api` instance group
1. ssh to the diego-api
1. `tail -f /var/vcap/sys/log/bbs/bbs.access.log | grep API-IP-ADDRESS | grep /v1/actual_lrp_groups`
1.  in another term window, ssh to the diego-api
1. `tail -f /var/vcap/sys/log/bbs/bbs.access.log | grep API-IP-ADDRESS | grep /v1/actual_lrps`
1. push an app
1. expect no results to appear in the term window running `tail...actual_lrp_groups` 
1. expect results to appear in the term window running `tail...actual_lrps`

**Pull request merged**